blockstack_client/config.py

```diff
@@ -987,62 +994,64 @@ def get_app_config(app_name, path=CONFIG_PATH):
 
 
 def update_config(section, option, value, config_path=CONFIG_PATH):
-
     parser = SafeConfigParser()
 
     try:
         parser.read(config_path)
-    except Exception, e:
+    except Exception as e:
         log.exception(e)
         return None
 
-    if parser.has_option(section, option):
-        parser.set(section, option, value)
+    if not parser.has_option(section, option):
+        return
+
+    parser.set(section, option, value)
+
+    with open(config_path, 'wb') as configfile:
+        parser.write(configfile)
+
 
-        with open(config_path, 'wb') as configfile:
-            parser.write(configfile)
+def get_version_parts(whole, func):
+    return [func(_.strip()) for _ in whole[0:3]]
 
 
-def semver_match( v1, v2 ):
+def semver_match(v1, v2):
     """
     Verify that two semantic version strings match:
     the major, minor, and patch versions must be equal.
```

Comment:
My bad; this should say 'the major and minor versions must be equal; the patch version is not considered' or the like.